shacl12-core/index.html

```diff
@@ -297,15 +297,7 @@ <h3>Terminology</h3>
 					</div>
 				</div>
 				<div class="def">
-					<div class="term-def-header">Binding, Solution</div>
-					<div>
-						A <dfn data-lt="bindings">binding</dfn> is a pair (<a data-cite="sparql12-query/#defn_QueryVariable">variable</a>, <a>RDF term</a>), consistent with the term's use in [[!sparql12-query]].
-					    A <dfn data-lt="solutions">solution</dfn> is a set of bindings, informally often understood as one row in the body of the result table of a SPARQL query.
-					    Variables are not required to be bound in a solution.
-					</div>
-				</div>
-				<div class="def">
-				<div class="term-def-header">SHACL Subclass, SHACL superclass</div>
+					<div class="term-def-header">SHACL Subclass, SHACL superclass</div>
```

Comment:
Why uppercase `S` on `Subclass`, and lowercase `s` on `superclass`? It seems like these should be the same, but I'm not sure which... I'm *guessing* lowercase, based on line 302.
```suggestion
					<div class="term-def-header">SHACL subclass, SHACL superclass</div>
```

Reply:
Will be addressed in this PR because this one here is already closed. https://github.com/w3c/data-shapes/pull/705